client/lib/analytics/test/index.js

```diff
@@ -147,4 +153,86 @@ describe( 'Analytics', () => {
 			expect( window._tkq.push ).toHaveBeenCalledWith( [ 'identifyUser', 8, 'eight' ] );
 		} );
 	} );
+	describe( 'initialize', () => {
+		beforeEach( () => {
+			loadScript.mockReset();
+
+			cookie.parse.mockImplementation( () => ( {} ) );
+			cookie.serialize.mockImplementation( () => {} );
+		} );
+		test( 'if stat.js load fails, should load nostat.js', () => {
```

Comment:
No need to change anything here, but if you don't feel like using `Promise.then` you could use `async () => {`, then 

```
await analytics.initialize();
expect( loadScript ).toHaveBeenCalledWith( expect.stringMatching( /\/nostats.js/ ) );
```